core/src/test/java/brooklyn/catalog/internal/CatalogScanTest.java

```diff
@@ -161,13 +162,13 @@ public void testMoreTypesThanAnnotationsForApps() {
     
     @Test
     public void testAnnotationIsDefault() {
-        loadAnnotationsOnlyCatalog();
         loadTheDefaultCatalog();
+        int numInDefault = Iterables.size(defaultCatalog.getCatalogItems(Predicates.alwaysTrue()));
         
+        loadAnnotationsOnlyCatalog();
         int numFromAnnots = Iterables.size(annotsCatalog.getCatalogItems(Predicates.alwaysTrue()));
-        int numInDefault = Iterables.size(defaultCatalog.getCatalogItems(Predicates.alwaysTrue()));
         
-        Assert.assertEquals(numFromAnnots, numInDefault);
+        Assert.assertEquals(numInDefault, numFromAnnots);
```

Comment:
Could also assert that `> 0`; otherwise could be a bug and a coincidence that both are zero?